content/en/agent/guide/how_remote_config_works.md

```diff
@@ -0,0 +1,149 @@
+| APM (feature: Dynamic Instrumentation) |               | 1.5.0         | 2.22.0        |               |
+| ASM feature: Protection                | 1.45.1        | 1.4.0         | 2.16.0        | 3.11.0        |
+| ASM feature: 1-click activation        |               | 1.4.0         | 2.17.0        | 3.9.0         |
+
+#### Setup:
+Once you've ensured that you meet the prerequisites, you can proceed with enabling Remote Configuration using the following steps: 
+
+1.  To authenticate and authorize your Agent to use Remote Configuration, you need to add Remote Configuration scope on your Datadog API key. [Contact support](https://docs.datadoghq.com/help/) to have your Organization added for Remote Config capability on API key.
+
+2. Ensure your RBAC permissions include [api_keys_write](https://docs.datadoghq.com/account_management/api-app-keys/) so you can add Remote Configuration scope on an existing API key, or create a new API key. Contact your organization's Datadog administrator and update your permissions if you don't have it.
+
+3. In your [Organization Settings](https://app.datadoghq.com/organization-settings/api-keys) page, either click on your existing API key, or click to Create a new API key.
+
+4. After creating a new API key, or clicking to edit your existing API key, you will see a screen to enable Remote Configuration:
+
+{{<img src="agent/guide/RC_Key_updated.png" alt="RC Key Updated" width="90%" style="center">}}
+
+5. Update your agent configuration file:
+
+{{< tabs >}}
+{{% tab "Configuration yaml" %}}
+Add the following to your configuration yaml.
+Ensure your Agent configuration is using the `api_key` that has Remote Config scope enabled. 
+```yaml
+api_key: xxx
+remote_configuration:
+  enabled: true
+  ```
+
+6. Restart your Datadog Agent for the changes to take effect.  
+
+{{% /tab %}}
+{{% tab "Environment variable" %}}
+Add the following to your Datadog Agent manifest.
+Ensure your Agent configuration is using the `api_key` that has Remote Config scope enabled. 
+```yaml
+DD_API_KEY=xxx
+DD_REMOTE_CONFIGURATION_ENABLED=true
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+With this configuration, your Agent requests its configuration from Datadog, and the features that use remote configuration are enabled:
+- CWS [default agent rules](https://docs.datadoghq.com/security/default_rules/#cat-workload-security) update automatically as released.
+- APM Agent-level [sampling rates](https://docs.datadoghq.com/tracing/trace_pipeline/ingestion_controls/#managing-ingestion-for-all-services-at-the-agent-level) are applied.  
+- [Dynamic Instrumentation](https://docs.datadoghq.com/dynamic_instrumentation/?tab=configurationyaml#enable-remote-configuration) is enabled.
+- [ASM](https://docs.datadoghq.com/security/application_security/how-appsec-works/#built-in-protection) 1-Click enablement, IP blocking, and attack pattern updates are enabled.
```

Comment:
```suggestion
- [CWS default agent rules][5] update automatically as released.
- [APM Agent-level sampling rates][6] are applied.  
- [Dynamic Instrumentation][7] is enabled.
- [ASM 1-Click enablement, IP blocking, and attack pattern updates][8] are enabled.
```
`[5]: /security/default_rules/#cat-workload-security`
`[6]: /tracing/trace_pipeline/ingestion_controls/#managing-ingestion-for-all-services-at-the-agent-level`
`[7]: /dynamic_instrumentation/?tab=configurationyaml#enable-remote-configuration`
`[8]: /security/application_security/how-appsec-works/#built-in-protection`